File: AdventJS/2025/reto 3/3.js
function drawGift(size, symbol) {
	if (size < 2) {
		return '';
	}

	let envelope = "";
	envelope += `${symbol.repeat(size)}`;
	for (let i = 0; i < size - 2; i++) {
		envelope += `\n${symbol}${" ".repeat(size - 2)}${symbol}`;
	}
	envelope += `\n${symbol.repeat(size)}`;

	return envelope;
}

File: AdventJS/2025/reto 3/3.py
def draw_gift(size, symbol):
  if size < 2:
    return ''

  envelope = ""
  envelope += symbol*size
  for _ in range(size - 2):
    envelope += f"\n{symbol}{" " * (size - 2)}{symbol}"
  envelope += f"\n{symbol*size}"
  return envelope